tuxemon/core/components/event/actions/combat.py

```diff
@@ -213,6 +213,9 @@ def random_encounter(self, game, action):
         if not self.check_battle_legal(player1):
             return False
 
+        #stop player motion
+        player1.moving = False
```

Comment:
So could we set player.moving to False then round the current player.tile_pos back to were it should be?